package.json

```diff
@@ -49,85 +50,70 @@
     "release:ci:major": "release-it major --ci"
   },
   "dependencies": {
-    "live-connect-common": "^1.0.0",
+    "live-connect-common": "file:../live-connect-common/live-connect-common-3.0.0.tgz",
+    "live-connect-handlers": "file:../live-connect-handlers/live-connect-handlers-3.0.0.tgz",
```

Comment:
Will be set to a proper dependencies when `live-connect-common` and `live-connect-handlers` are published.